Sources/BrightroomEngine/Core/EditingStack.swift

```diff
@@ -57,6 +57,12 @@ open class EditingStack: Equatable, StoreComponentType {
     lhs === rhs
   }
 
+  public func hash(into hasher: inout Hasher) {
+    withUnsafePointer(to: self) {
+      hasher.combine($0)
+    }
+  }
```

Comment:
how about using just `ObjectIdentifier`?

Reply:
public func hash(into hasher: inout Hasher) {
    ObjectIdentifier(self).hash(into: &hasher)
  }

just like this